.eslintrc

```diff
@@ -43,7 +43,6 @@
     "no-empty-character-class": 2,
     "no-ex-assign": 2,
     "no-extra-boolean-cast": 2,
-    "no-extra-parens": 0,
```

Comment:
no-extra-parens is found twice in that file. `"no-extra-parens": 0` and `"no-extra-parens": [2, "functions"]`. The upgrade of grunt-eslint from 17.1.0 -> 19.0.0 identified this.